src/by.jl

```diff
@@ -1,8 +1,9 @@
 """
     by(formula, by, design, function, params)
 
-Generate subsets of a survey design.
+Estimate the population parameters of for subpopulations of interest for a simple random sample.  
 
+In the following example, the mean `api00` is estimated for each county. 
```

Comment:
This sentence can be removed. It is already in the documentation and docstrings don't need these explanations, I think.

Reply:
We'll be removing `by`.